docs/docs-content/integrations/ngrok.md

```diff
@@ -0,0 +1,142 @@
+---
+sidebar_label: "ngrok"
+title: "ngrok"
+description: "Learn about using ngrok Kubernetes Ingress to access applications and services in Spectro Cloud."
+hide_table_of_contents: true
+type: "integration"
+category: ["ingress", "kubernetes"]
+sidebar_class_name: "hide-from-sidebar"
+logoUrl: ""
+tags: ["packs", "ngrok", "network", "kubernetes"]
+---
+
+The ngrok [Ingress Controller for Kubernetes](https://github.com/ngrok/kubernetes-ingress-controller) is the official
+controller for adding public and secure ingress traffic to your k8s services. This open source [Ingress
+Controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers) works with Spectro Cloud to
+provide ingress to your applications, APIs, or other services while also offloading network ingress and middleware
+execution to ngrok's platform.
+
+# Versions Supported
+
+- 0.9.0
+
+## Prerequisites
+
+- An active [ngrok account](https://ngrok.com/signup).
+  - Your ngrok authentication token (`AUTHTOKEN`), which you can find under under [**Your
+   Authtoken**](https://dashboard.ngrok.com/get-started/your-authtoken) in the ngrok dashboard.
+  - An API key (`API_KEY`), which you can generate in the [**API** section](https://dashboard.ngrok.com/api) of the
+    ngrok dashboard.
+- A static subdomain (`NGROK_DOMAIN`), which you can obtain by navigating to the [**Domains**
+   section](https://dashboard.ngrok.com/cloud-edge/domains) of the ngrok dashboard and clicking **Create Domain** or
+   **New Domain**.
+
+## Parameters
+
+To deploy properly, the ngrok Kubernetes Ingress Controller requires your ngrok authentication `API_KEY` and
+`AUTHTOKEN`, which you can add to your preset or running profile.
+
+```yaml
+charts:  
+  kubernetes-ingress-controller:
+    credentials:
+      apiKey: API_KEY
+      authtoken: AUTHTOKEN
+```
+
+:::tip
+
+Free ngrok accounts can have only one ngrok agent running at a time. If you have a free ngrok account, change
+the number of replicas to `1`, like the following:
+
+```yaml
+charts:
+  kubernetes-ingress-controller:
+    replicaCount: 1
+```
+
+:::
+
+## Usage
```

Comment:
@joelhans I like how you included an example but I think we need to explain a bit more what is happening here. This section should focus on explaining the `ingress` section of the YAML and why the current values are used.

 Think of this section as helping the reader understand how to use the ngrok controller pack. Don't duplicate information, link to existing resources that you all have created, but do go into some details of what is happening here so the reader can understand.